test/scripts/filters/backtick_code_block.js

```diff
@@ -1,6 +1,8 @@
-const should = require('chai').should(); // eslint-disable-line
+'use strict';
+
+require('chai').should();
```

Comment:
Even if we abandon the return value that calls `should()` like the sample code of https://www.chaijs.com/guide/styles/#using-should-in-es2015, in principle no problem.
By calling `should()`, the should property is added to `Object.prototype` via `Object.defineProperty()`. You can use should from almost any value after that.
However, since only `null`, `undefined`, or prototype null objects can not call `Object.prototype` from the prototype chain, you can not get `should` property.
The return value of `should()` is used to assert such as `should.not.exist(undefined)` for these values.